python/vyos/validate.py

```diff
@@ -239,6 +239,27 @@ def assert_mac(m):
     if octets[:5] == (0, 0, 94, 0, 1):
         raise ValueError(f'{m} is a VRRP MAC address')
 
+def has_vlan_subinterface_configured(conf, intf):
```

Comment:
This should be moved to `configdict.py` as this is a similar function as `is_member()`.